src/clusterfuzz/_internal/bot/fuzzers/libfuzzer.py

```diff
@@ -1459,8 +1459,11 @@ def pick_strategies(strategy_pool,
 
   # Do not use fork mode for DFT-based fuzzing. This is needed in order to
   # collect readable and actionable logs from fuzz targets running with DFT.
+  # If fork_server is already set by the user, let's keep it that way.
+  fork_server = fuzzer_utils.extract_argument(
+      existing_arguments, constants.FORK_FLAG, remove=False)
   if (not is_fuchsia and not is_android and not is_ephemeral and
-      not use_dataflow_tracing and
+      not use_dataflow_tracing and not fork_server and
```

Comment:
shouldn't this be `and fork_server` not `and not fork_server`?

Reply:
No. Basically, it says: "If the options wasn't set by the user, go set it".